sympy/utilities/iterables.py

```diff
@@ -19,6 +19,40 @@
     multiset_partitions_taocp, list_visitor, MultisetPartitionTraverser)
 
 
+def is_palindromic(s, i=0, j=None):
```

Comment:
I don't think that there is a usage of `i`, `j` keywords. Slicing can always be done priori, so I don't think that it hurts any generalization without these keywords.

Reply:
I am avoiding having to create a new sequence by allowing the test to be done in place. The call to this routine from `is_palindromic` in `digits` uses the `digits` result which needs to be tested from position 1 through the end.
```python
>>> timeit('[1,2,3,4,5][1:3][0]')
0.5159061519952957
>>> timeit('[1,2,3,4,5][0]')
0.23576541400689166
```